hedera-node/docs/design/services/smart-contract-service/IHFC-associate-dissociate.md

```diff
@@ -0,0 +1,81 @@
+# IHRC Facade Associate and Dissociate Functionality
```

Comment:
What does "IHRC" stand for?

Reply:
I believe we were mimicking IERC since that interface is familiar to people.  Hedera Request for Comment interface?

Reply:
Yeah open to naming if it's not obvious.
We want familiarity so ETH devs understand